Solving a practical task 4th sprint

r-yy opened the pull request.

r-yy committed:
Deleted console outputting message

r-yy committed:
Solving a practical task 4th sprint

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -2,11 +2,164 @@ import UIKit
+        showAnswerResult(isCorrect: false == currentQuestion().correctAnswer)
+    }
+    
+    @IBAction private func yesAction(_ sender: UIButton) {
```
Лучше называть yesButtonPressed и noButtonPressed, т.к в таком варианте присутствует глагол, что более корректно для методов.

r-yy replied:
Кнопки переименовал

r-yy committed:
1. Fix rounded corners at first time show imageView
2. Fix colors
3. Fix disable buttons between questions
4. Fix Launch Screen display on iPhone 14 Pro and 14 Pro Max
5. Fix status bar display

r-yy committed:
Delete MovieQuiz/Resources/Assets.xcassets/Colors/Black.colorset directory

r-yy committed:
Delete MovieQuiz/Resources/Assets.xcassets/Colors/Gray.colorset directory

r-yy committed:
Delete MovieQuiz/Resources/Assets.xcassets/Colors/Green.colorset directory

r-yy committed:
Delete MovieQuiz/Resources/Assets.xcassets/Colors/Red.colorset directory

r-yy committed:
Delete MovieQuiz/Resources/Assets.xcassets/Colors/White.colorset directory
